src/Quantities.purs

```diff
@@ -24,9 +24,10 @@ import Data.Quantity (ConversionError(..), Quantity, abs, approximatelyEqual, as
                       isFinite, pow, prettyPrint, prettyPrint', qAdd, qDivide, qMultiply,
                       qNegate, qSubtract, quantity, quantity', scalar, scalar', showResult,
                       sqrt, toScalar, toScalar', toStandard, (.*), (⊕), (⊖), (⊗), (⊘)) as DQ
-import Data.Quantity.Math (acos, acosh, asin, asinh, atan, atan2, atanh, ceil, cos, cosh, e,
-                           exp, factorial, floor, gamma, ln, log10, max, max2, mean, min,
-                           min2, modulo, pi, round, sin, sinh, tan, tanh, tau) as DQM
+import Data.Quantity.Math (asec, acsc, acot, sec, csc, cot, csch, sech, coth, acsch, asech,
```

Comment:
I know this is supposed to be ordered alphabetically, but it was such a big pain to put each function in its proper place that I gave up in the end.

Reply:
Nevermind, figured it out.

Reply:
Sorry about that. For future reference: I don't care too much about the ordering of those inputs :smile:

Reply:
I just ended up doing some Vim voodoo in the end. :D

Reply:
vim voodoo. nice. I'm going to borrow that phrase to refer to my vim macro hacks in the future :smile: